lib/framework/view/bottom_nav_page_group.dart

```diff
@@ -99,6 +112,10 @@ class _BottomNavPageGroupState extends State<BottomNavPageGroup>
       floatingAlignment =
           FloatingAlignment.values.byName(fabMenuItem!.floatingAlignment);
     }
+
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
This is suboptimal. Why let it render, then re-render the correct page? Don't use this callback please.